spk/rabbitmq-server/src/service-setup.sh

```diff
@@ -0,0 +1,45 @@
+SERVICE_COMMAND='/var/packages/rabbitmq-server/target/lib/rabbitmq_server-3.8.12/sbin/rabbitmq-server'
+SVC_CWD="${SYNOPKG_PKGDEST}"
+SVC_BACKGROUND=y
+SVC_WRITE_PID=y
+
+
+
+service_preinst ()
+{
+    echo "service_preinst ${SYNOPKG_PKG_STATUS}" >> $INST_LOG
+}
```

Comment:
you must not implement empty functions here (and below)

Reply:
Ok it has been removed